src/createServer.ts

```diff
@@ -3,89 +3,73 @@ import {
   RequestPackageOptions,
   WantedDependency,
 } from '@pnpm/package-requester'
-import JsonSocket = require('json-socket')
-import net = require('net')
+
+import bodyParser = require('body-parser')
+import {Request, Response} from 'express';
+import express = require('express')
+import http = require('http')
+
 import {StoreController} from 'package-store'
 
 export default function (
   store: StoreController,
-  opts: object,
+  opts: {
+    path: string;
+  },
 ) {
-  const server = net.createServer()
-  server.listen(opts)
-  server.on('connection', (socket) => {
-    const jsonSocket = new JsonSocket(socket)
-    const requestPackage = requestPackageWithCtx.bind(null, {jsonSocket, store})
+  const packageResponses = {}
+
+  const app = express()
+  app.use(bodyParser.json({
+    limit: '10mb',
+  }))
+
+  app.post('/requestPackage', async (request: Request, response: Response) => {
```

Comment:
instead of the separate actions, it might be faster to have a single route with a switch inside.